### ISSUE TITLE
[Accessibility]: Batches: Screenreader doesnt specify editor is a text area

### ISSUE THREAD

**Issue by adeola-ak** [sourcegraph/sourcegraph#34832]
### Audit type

Screen reader navigation

### User journey audit issue

#34078

### Problem description

After Batch change is created, and page navigates from Create to Edit. It begins reading the Editor but doesnt specify that it is a text field.

### Expected behavior

Monaco Editor is specified as a text area

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by adeola-ak** [comment]
determined this was okay, as long as the text gets read by the screenreader which it does
